src/utils.py

```diff
@@ -34,7 +34,9 @@ def ensure_parent_containers(tiled_uri, tiled_api_key):
 
     last_container = from_uri(tiled_root, api_key=tiled_api_key)
 
-    container_parts = path_parts[3:]
+    container_parts = path_parts[
```

Comment:
Assume we have not specified a subpath but are just writing to the base uri http://localhost:8000/, does the function here still do what we want it to do? 
It seems we would run into errors since this assumes that path_parts will have at least 4 elements?

Reply:
You are right. With the case you suggested above, this will fail. I recall from last Friday's discussion, the `api/v1/metadata` will always be the pre-existing part once a tiled client is installed and ready for the ingestion. And I believe we made a decision of going forward with this assumption due to the limited time.

We can revisit this again later, but having `[3:]` here will prevent the duplication of the above path for the current MVP.